docker/linux/base/build/Dockerfile

```diff
@@ -33,23 +36,24 @@ RUN apt update -qqy \
 ARG NVM_SHA256SUM="f068e17dacb88f73302790cc076956c7a0d459ce9b01df842ff3e75744f9e2fe /tmp/install.sh"
 ARG NVM_URL="https://cobalt.googlesource.com/third_party/nvm/+/refs/tags/v0.35.3/install.sh?format=TEXT"
 ENV NVM_DIR=${HOME}/.nvm
-ENV NODE_VERSION 16.19.1
-
+ 
+ENV NODE_VERSION=
 RUN curl --silent -o- ${NVM_URL} \
   | base64 -d > /tmp/install.sh \
    && echo ${NVM_SHA256SUM} | sha256sum --check \
    && . /tmp/install.sh
 
 # === Pinned Node version to v16, as this is latest one that will work on
 # legacy Debian systems
-ARG LTS_VERSION=v16
+ARG NODE_LTS_VERSION=lts/gallium
```

Comment:
Please do not change the currently hardcoded defaults for versions. It's okay to make them properly parametrized if this is needed for some reason, e.g. `ARG` with `ENV` default, like FROM_IMAGE at the top of this file. I.e.
```
ARG NODE_VERSION
ENV NODE_LTS_VERSION=${NODE_VERSION:-v16}
```
This would leave the default unchanged, but allow parameterization via docker-compose invocation.